shell/edit/provisioning.cattle.io.cluster/Basics.vue

```diff
@@ -413,6 +420,12 @@ export default {
         v-clean-html="t('cluster.harvester.warning.cloudProvider.incompatible', null, true)"
       />
     </Banner>
+    <Banner
+      v-if="showCloudProviderAmazonAdditionalConfigWarning"
+      color="warning"
+    >
+      <span v-clean-html="t('cluster.banner.cloudProviderAddConfig', {}, true)" />
```

Comment:
It doesn't look like the translation string uses any arguments so I'm not sure you need the second and third parameter on the `t` function here.

Reply:
It needs the last arg, so that it renders the HTML (anchor) that the translation contains. Usually the second arg is `null` but an empty object also works.